Added disclaimer (#1062)
demisto/content-docs: content-repo/extra-docs/articles/troubleshooting.md

## Debug Mode
Cortex XSOAR (Server 5.0+) supports running Python integration commands and automation scripts in `debug-mode` from the Cortex XSOAR CLI. When a command is run in `debug-mode` a log file of the command execution will be created and attached to the war room. When encountering an issue which is related to an integration or an automation, make sure to reproduce the command with `debug-mode` and inspect the generated log file. The `debug-mode` log file will contain information not available in the Server logs and can provide additional insights regarding the root cause of the issue. Additionally, some integrations have specific code to include extra debug info when run in `debug-mode`.

:::caution Important Note

> Debug mode feature prints extended data from integrations configuration and settings which might include sensitive information. Before sharing these files make sure your secretes are removed.

Reviewer: ```suggestion
The debug mode feature prints extended data from an integrations configuration and settings which may include sensitive information. Before sharing these files, make sure sensitive information has been removed.
```
